File: design-hashset/design-hashset.java
class MyHashSet {
    int[] s;
    public MyHashSet() {
        s = new int[1000000+1];
    }
    public void add(int key) {
        s[key] = 1;
    }
    public void remove(int key) {
        s[key] = 0;
    }
    public boolean contains(int key) {
        return s[key]==1;
    }
}

// Design hashset

File: design-hashset/design-hashset.py
class MyHashSet:
    def __init__(self):
        self.s = [0]*(10**6 + 1)
        
    def add(self, key: int) -> None:
        self.s[key] = 1

    def remove(self, key: int) -> None:
        self.s[key] = 0

    def contains(self, key: int) -> bool:
        return self.s[key] == 1


# design hashset